labs/lab3/chapter3.md

```diff
@@ -212,14 +212,16 @@ $ curl -L http://localhost:8080 #note we indicated the port to use in the run co
   **Note**: the `curl` command returns an error but demonstrates
             a response on the port.
 
-5. Test the Wordpress image to confirm connectivity. For the mariadb container we need to specify an additional option to make sure it is in the same "network" as the apache/wordpress container and not visible outside that container:
+5. Bring up the database (mariadb) for the wordpress instance. For the mariadb container we need to specify an additional option to make sure it is in the same "network" as the apache/wordpress container and not visible outside that container:
+
   * `--network=container:<alias>` to link to the wordpress container
 ```bash
 $ ls -lZd ~/workspace/mysql
 $ sudo podman run -d --network=container:wordpress -v ~/workspace/mysql:/var/lib/mysql:z -e DBUSER=user -e DBPASS=mypassword -e DBNAME=mydb --name mariadb mariadb
 ```
 Note: See the difference in SELinux context after running w/ a volume & :z.
 ```bash
+$ ls -lZd ~/workspace/mysql
 $ ls -lZ ~/workspace/mysql
```

Comment:
do we really want both of these?

Reply:
i don't think so

Reply:
the problem here is that on line 218/219 above we `ls -lZd ~/workspace/mysql`. If we don't show that the label on the dir changed then we aren't showing much?

Reply:
I just feel like the selinux stuff is a bit of a branch anyway. As long as they are different enough to be interesting.. sure, have both.. doesn't hurt